tests/parser/tests/cases/should_pass/unary_ops/case.hash

```diff
@@ -0,0 +1,3 @@
+let k = ~a;
+let k = ---a;
```

Comment:
In the future, we probably want a lint to say "Hey, --a does not mean to decrement a!".

Reply:
that's a good point, it should be explicitly stated in the docs that it doesn't do what you think it does.

Reply:
Maybe an issue can be created for that.